backtrader/feeds/psqlfeed

```diff
@@ -0,0 +1,86 @@
+# !/usr/bin/env python
+# -*- coding: utf-8; py-indent-offset:4 -*-
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from backtrader import DataBase, date2num
+import datetime as dt
+import urlparse
+import psycopg2
+
+class PsqlDatabase(DataBase):
+    '''
+    Fetches data from PSQL, and wraps it into a Feed consumable by cerebro
+    takes url connection string in form of :
+        postgres://{user}:{pw}@{host}:{port [optional]}/{database}
+
+    must also pass a datestart and a table name conforming to a similar schema as following:
+            ticker        char(5),
```

Comment:
Are you assuming the table schema to include the symbol name - meaning the same table to include the data for all symbols?  

What if user prefer to keep each symbol data in a separate table? or even separate table for each timeframe per symbol?  

What should be done if multiple timeframe data should be stored?

Reply:
this implementation is with all symbols in one table, keyed with date and symbol.
one could implement it differently in the ways you mentioned. afaik it would perform the exact same. 
the interface is going to be bespoke to the way the database is set up.

same goes for dup date entries. i think its a reasonable assumption not to have duplicate entries. same would go for a csv. i think that assumption is reflected in the "...unique(date, symbol)..." part of the schema definition. 

one could just as well have duplicate entries or swap out date for datetime, and have some logic to pick an entry. i feel that's out of the scope of this PR :)

Reply:
> afaik it would perform the exact same

IIRC, indexes are kept per table in PSQL. Although index size is not growing linearly with the table size ( let's take a B-Tree for example) - its performance will decrease as table size grows.

> the interface is going to be bespoke to the way the database is set up

ok - it is reasonable. So if one decides to have a different scheme and inherits this class, the `start` method needs to be reimplemented, right?

Reply:
yes that's correct. i've added a comment noting this.